edb/lib/std/25-numoperators.eql

```diff
@@ -378,14 +406,33 @@ std::`>` (l: std::float64, r: std::float64) -> std::bool
 
 
 CREATE INFIX OPERATOR
-std::`>` (l: std::anyreal, r: std::decimal) -> std::bool
+std::`>` (l: std::decimal, r: std::decimal) -> std::bool
+    FROM SQL OPERATOR r'>';
+
+
+CREATE INFIX OPERATOR
+std::`>` (l: std::float64, r: std::decimal) -> std::bool
+    FROM SQL $$
+        SELECT "l" > "r";
+    $$;
```

Comment:
Please use this formatting style:

```
{
     FROM SQL $$
     SELECT round("val")
     $$;
 };
```

Reply:
I've updated indentation to be consistent.